File: src/main/java/com/melonbase/microquark/repo/data/Vorlage.java
package com.melonbase.microquark.repo.data;

public class Vorlage {

  private String beschreibung;
  private Wahlresultat wahlresultat;

  public Vorlage(String beschreibung) {
    this.beschreibung = beschreibung;
  }

  public String getBeschreibung() {
    return beschreibung;
  }

  public Wahlresultat getWahlresultat() {
    return wahlresultat;
  }

  public void setWahlresultat(Wahlresultat wahlresultat) {
    this.wahlresultat = wahlresultat;
  }
}

File: src/main/kotlin/com/melonbase/microquark/repo/data/Vorlage.kt
package com.melonbase.microquark.repo.data

import one.microstream.reference.Lazy

class Vorlage(var beschreibung: String) {

  private var lazyWahlresultat: Lazy<Wahlresultat> = Lazy.Reference(null)

  fun getWahlresultat(): Wahlresultat? = Lazy.get(lazyWahlresultat)

  fun setWahlresultat(wahlresultat: Wahlresultat) {
    lazyWahlresultat = Lazy.Reference(wahlresultat)
  }
}